### ISSUE TITLE
"Any of" not working

### ISSUE THREAD

**Issue by archdruid101** [AngryBeaver/beavers-crafting#89]
The logic given on the readme does not seem to work anymore

**Comment by archdruid101** [comment]
It seems that the Recent update to have subtype sources messed with it a bit

**Comment by AngryBeaver** [comment]
Yes you are partly right the example is outdated in the current version of dnd5e:

In AnyOf you can define a macro method that returns a boolean (true/false). The macro will then be executed to any item in your inventory or when dropping a specific item on it only to this. The "item" variable will hold a current reference to the tested item **the document structure may change on diffrent system as each system defines item for it self**

If you do not know exaclty how your system item structure is looking like you can always just print it to console (hit f12).
write an anyof with this: console.log(item); return false; this will never accept anything but logs the document structure of your 
item.

![image](https://github.com/AngryBeaver/beavers-crafting/assets/10677192/fcc161bf-ed8b-437a-a4be-15f405086621)
so the latest dnd5e system has apparently updated the item structure. But anyway this is only an example and on top only one that would asume you use a specific system. I will update this next but no need to stick exaclty to my old macros :-).

return item.name.startsWith("test") just use any attribute you see fit.

**Comment by AngryBeaver** [comment]
readme adapted
